src/collective/cover/layout.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from collective.cover.grid import BaseGrid  # noqa
+from zope import deprecation
+
+
+deprecation.deprecated(
```

Comment:
I copied this code from https://github.com/plone/plone.dexterity/blob/8d51adeed2070b23813f0ba2df54aed8c1baab0a/plone/dexterity/utils.py#L28-L39